Mathlib/MeasureTheory/Function/StronglyMeasurable/Basic.lean

```diff
@@ -1883,6 +1883,12 @@ theorem _root_.aestronglyMeasurable_withDensity_iff {E : Type _} [NormedAddCommG
     simpa only [NNReal.coe_eq_zero, Ne.def] using h'x
 #align ae_strongly_measurable_with_density_iff aestronglyMeasurable_withDensity_iff
 
+lemma aestronglyMeasurable_of_absolutelyContinuous {α β : Type _} [MeasurableSpace α]
```

Comment:
This is exactly [MeasureTheory.AEStronglyMeasurable.mono'](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Function/StronglyMeasurable/Basic.html#MeasureTheory.AEStronglyMeasurable.mono') right?

Reply:
Yes, thanks! Deleted and updated.